pkg/codegen/go/utilities.go

```diff
@@ -102,3 +103,7 @@ func makeSafeEnumName(name, typeName string) (string, error) {
 
 	return safeName, nil
 }
+
+func camel(s string) string {
```

Comment:
Can we just fix up the code to call cgstrings.Camel directly

Reply:
Sure, there were just two dozen or so instances of `camel()` so the change was just smaller this way

Reply:
Done in https://github.com/pulumi/pulumi/pull/10738/commits/b034765b08d9d333e3c7f299b6399fd0aab9c128